boot/freeldr/freeldr/arch/i386/multiboot.S

```diff
@@ -139,32 +128,43 @@ mb1:
     jnz mb1
 
 mb2:
+#ifndef _M_IX86
+/* If the compiler is 64 bits then skip the 32 bits prefix we are already in protected mode */
+    .byte HEX(E9)
+    .long 1
+#endif
```

Comment:
What is this? The comment doesn't explain anything.

Reply:
it's a near jump that skip 1 byte of the instruction below.